panel/models/vtk.py

```diff
@@ -31,6 +31,8 @@ class VTKPlot(HTMLBox):
 
     camera = Dict(String, Any)
 
+    keybinds = Bool(default=True)
```

Comment:
Since this causes such issues let's default to False and call it ``keybinding``, or maybe @jbednar has a better suggestion for naming?

Reply:
I agree it should be false by default; people can enable it in a standalone dashboard but it seems very unlikely to be useful in a notebook until it can be made to respect focus.  Seems like it could have a very descriptive name like `capture_keypress_events`; it won't come up often and something like that will be very clear.

Reply:
``enable_keybindings``?

Reply:
That's fine, but to me it doesn't convey that it will be listening to all keypresses.  Whereas if someone explicitly enables `capture_keypress_events`, they surely won't come complaining to us when they see it doing just that. :-)  But I'll leave it to your discretion.

Reply:
I think that's okay for an option you have to enable explicitly, as long as it's properly documented that is.

Reply:
@xavArtley Sorry to have been confusing here, I think ``enable_keybindings`` is the best name for the parameter/property.